scripts/update-patch.sh

```diff
@@ -9,8 +9,8 @@ pushd "$SCRIPTS_DIR/../playwright"
 SCRIPTS_DIR="$(dirname "$0")"
 echo "Creating patch..."
 git add .
-git diff --full-index playwright-build^1..playwright-build > ../patches/main.patch
+git diff --full-index 396487f..main > ../patches/main.patch
```

Comment:
this looks not how it should be.

Reply:
yeah this part I didn't understand that's why I asked you how to create the patch and then use it

Reply:
I fixed the tests all and the test works with `1.25.2` version, I am not able to get the types/interface generation part